src/python/pants/releases/packages.py

```diff
@@ -75,23 +81,118 @@ def owners(self,
     return {owner.lower() for owner in owners}
 
 
+def find_platform_name():
+  # See: https://www.python.org/dev/peps/pep-0425/#id13
+  return get_platform().replace("-", "_").replace(".", "_")
+
+
 core_packages = {
-  Package("pantsbuild.pants"),
-  Package("pantsbuild.pants.testinfra"),
+  Package(
+    "pantsbuild.pants",
+    "//src/python/pants:pants-packaged",
+    bdist_wheel_flags=("--python-tag", "cp27", "--plat-name", find_platform_name()),
+  ),
+  Package("pantsbuild.pants.testinfra", "//tests/python/pants_test:test_infra"),
 }
 
 
 def contrib_packages():
-  output = subprocess.check_output(
-    ('bash', '-c', 'source contrib/release_packages.sh ; for pkg in "${CONTRIB_PACKAGES[@]}"; do echo "${!pkg}"; done')
-  ).decode('utf-8')
-  return {Package(name) for name in output.strip().split('\n')}
+  return {
+    Package(
+      "pantsbuild.pants.contrib.scrooge",
+      "//contrib/scrooge/src/python/pants/contrib/scrooge:plugin",
+    ),
```

Comment:
I think you should also update `contrib/README.md` where it talks about updating these files. It's near
`To register with the release script, add an entry to contrib/release_packages.sh:`